bugsnag nullPointerException issue，any body can help me?

Caused by: java.lang.NullPointerException
at com.bugsnag.Error.getExceptionName(Error.java:198)
at com.bugsnag.Error.shouldIgnore(Error.java:96)
at com.bugsnag.Client.notify(Client.java:115)
at com.bugsnag.Client.notify(Client.java:128)
at com.bugsnag.Client.notify(Client.java:136)
at com.huawei.bes.common.log.impl.BesLogImpl.debug(BesLogImpl.java:206)
at com.huawei.bes.common.log.impl.BesLogImpl.debug(BesLogImpl.java:178)
at com.huawei.bes.common.dfx.monitor.BESCustomEBusStatistic.initFilter(BESCustomEBusStatistic.java:86)
at com.huawei.bes.common.dfx.monitor.BESCustomEBusStatistic.init(BESCustomEBusStatistic.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:483)
at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeCustomInitMethod(AbstractAutowireCapableBeanFactory.java:1581)

--- a/src/main/java/com/bugsnag/Client.java
+++ b/src/main/java/com/bugsnag/Client.java
@@ -118,9 +118,13 @@ public class Client {
     }
 
     public void notify(Error error) {
-        if (!config.shouldNotify()) return;
-        if (error.shouldIgnore()) return;
-        if (!beforeNotify(error)) return;
+        if (error == null || error.getException() == null) {
+            config.logger.warn("Report not sent to Bugsnag, Throwable is null");
+            return;
+        }
+        if (!config.shouldNotify() ||
+            error.shouldIgnore() ||
+            !beforeNotify(error)) return;
 
         Notification notif = new Notification(config, error);
         if (config.asynchronousNotification) {

